playbook.yml

```diff
@@ -7,6 +7,72 @@
       assert:
         that: druid_s3_accessKey != '<AWS_ACCESS_KEY>' and druid_s3_secretKey != '<AWS_SECRET_KEY>'
 
+- hosts: all
+  become: yes
+  gather_facts: yes
+  vars_files:
+    - vars/druid.yml
+  roles:
+    - java
+
+- hosts: tag_metrics_master
+  vars_files:
+    - vars/metrics.yml
+  become: yes
+  roles:
+    - influxdb
+    - grafana
+
+  post_tasks:
+    - name: Install influxdb python module
+      pip:
+        name: influxdb
+
+    - name: Create influxdb database for druid metrics
+      influxdb_database:
+        hostname: "{{ ansible_fqdn | default(ansible_host) | default(inventory_hostname) }}"
+        database_name: druid
+
+    - name: Enable graphite in package-provided config
+      blockinfile:
+        dest: "{{ influxdb_config_file }}"
+        marker: "  # {mark} ANSIBLE MANAGED GRAPHITE ENABLEMENT BLOCK"
+        insertafter: ^\[\[graphite\]\]
+        block: |2
+            enabled = true
+            database = "druid"
+            templates = [
+              "*.*.*.*.*.* .service.hostname.dim1.dim2.measurement*",
```

Comment:
At ING, since we use one grafana/influx for different environments I made sure that the `env` is also listed (it's the first match of the filter). So preferably we record it, doesn't do any harm but potentially very useful. So suggested change would be:

`"*.*.*.*.*.* env.service.hostname.dim1.dim2.measurement*",`

and same for the other lines